src/DIPS.Xamarin.UI/Controls/Popup/PopupBehavior.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Extensions;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Popup
+{
+    /// <summary>
+    /// Behavior to be added to an item you want to open a popup from. This item _has_ to be inside a PopupLayout somehow. Else, nothing would happend.
+    /// </summary>
+    [ContentProperty(nameof(Content))]
+    public class PopupBehavior : Behavior<View>
+    {
+        private readonly Command m_onTappedCommand;
+        private View? m_attachedTo;
+        /// <summary>
+        /// Creates the instance
+        /// </summary>
+        public PopupBehavior()
+        {
+            m_onTappedCommand = new Command(ShowPopup);
+        }
+
+        /// <summary>
+        /// Handels attaching to item
+        /// </summary>
+        /// <param name="bindable"></param>
+        protected override void OnAttachedTo(View bindable)
+        {
+            m_attachedTo = bindable;
+            if (bindable is Button button)
+            {
+                button.Command = m_onTappedCommand;
+            }
```

Comment:
Can the consumer use its own command on the button? Maybe use the `Clicked` event instead?